docs/docusaurus.config.js

```diff
@@ -89,7 +89,7 @@ module.exports = {
         docs: {
           sidebarPath: require.resolve('./sidebars.js'),
           editUrl:
-            'https://github.com/anitab-org/bridge-in-tech-backend/docs',
+            'https://github.com/anitab-org/bridge-in-tech-backend/blob/develop/docs/docs/Home.md',
```

Comment:
```suggestion
            'https://github.com/anitab-org/bridge-in-tech-backend/edit/develop/docs/'
```
Also specifying the home page is wrong here as it will lead to only the home page or 404 because the slug is appended in edit URL by docusaurus for different sidebar slugs.